### ISSUE TITLE
Can we setup a single api and use it to make different endpoints req

### ISSUE THREAD

**Issue by Aadarsh805** [Aman-Mandal/groovy#89]
setup a baseURL: 'https://shazam.p.rapidapi.com/'
and make multiple endpoints like
getTopCharts will fetch from baseURL/topCharts
getTopArtists will fetch from baseURL/topArtists

**Comment by Aman-Mandal** [comment]
No idea Sir, but we can give it a shot

**Comment by diegolikescode** [comment]
yes we can @Aadarsh805, we will fetch the data from the Next app, right? U can assign this issue for me too and I'll do it <3

**Comment by Aadarsh805** [comment]
> yes we can @Aadarsh805, we will fetch the data from the Next app, right? U can assign this issue for me too and I'll do it <3

that's great you got it, good luck!!

**Comment by diegolikescode** [comment]
@Aadarsh805 dude, I have done and tested the rapidapi but these guys don't let you do that much requests, actually I think I already did my monthly request limit, wish to try some other music API? The axios config is done and we just need the right API

**Comment by Aman-Mandal** [comment]
> @Aadarsh805 dude, I have done and tested the rapidapi but these guys don't let you do that much requests, actually I think I already did my monthly request limit, wish to try some other music API? The axios config is done and we just need the right API

Exactly!! We are facing alot if issues with Shazam API..I think we need to change the API!! 

You know about any good API for music?

**Comment by diegolikescode** [comment]
not really, but I'll do some research and test somethings, the axios' config is already done, I'll do it boys

**Comment by Aadarsh805** [comment]
> not really, but I'll do some research and test somethings, the axios' config is already done, I'll do it boys

Thanks Diego it'd be really helpful
